src/redux/books/books.js

```diff
@@ -1,22 +1,28 @@
 const ADD_BOOK = 'books / books / ADD_BOOK';
 const REMOVE_BOOK = 'books / books / REMOVE_BOOK';
```

Comment:
- [ ] **Required:** Kindly remove the white spaces in your action string. For example `const ADD_BOOK = 'books / books / ADD_BOOK';` should be `const ADD_BOOK = 'books/books/ADD_BOOK';`. This is to ensure it complies with the duck pattern convention.